src/index.js

```diff
@@ -0,0 +1,111 @@
+import express from 'express';
+import { v4 as uuidv4 } from 'uuid';
+import _ from 'lodash';
+
+const PORT = Number(process.env.PORT) || 3000;
+const app = express();
+app.use(express.json());
+
+let users = [];
+const PROPERTIES = [
+    'login',
+    'password',
+    'age'
+];
+
+const findUser = (id) => users
+    .filter(user => !user.isDeleted)
+    .find(user => user.id === id);
+
+const addUser = (user) => {
```

Comment:
I'm almost sure that login has to be unique even though it's not mentioned in the task

Reply:
Decided not to do in chat